src/domain/chains/entities/rpc-uri-authentication.entity.ts

```diff
@@ -0,0 +1,4 @@
+export enum RpcUriAuthentication {
+  ApiKeyPath = 'API_KEY_PATH',
+  NoAuthentication = 'NO_AUTHENTICATION',
+}
```

Comment:
In the current CGW exists a third option `Unknown`

Reply:
Done in [34d40f2](https://github.com/5afe/safe-client-gateway-nest/pull/93/commits/34d40f226f67c3b006a3f1eccccfe9dc022ad746)